exasol/toolbox/nox/_release.py

```diff
@@ -89,6 +97,51 @@ def _add_files_to_index(session: Session, files: list[Path]) -> None:
         session.run("git", "add", f"{file}")
 
 
+def _type_release(release_type: ReleaseTypes, old_version: Version) -> Version:
```

Comment:
Thx for implementing the suggested change with `Version.upgrade_version_from_poetry`. We no longer need this method, so please remove 

- [ ] it
- [ ]  the tests associated with it